pilot/pkg/bootstrap/istio_ca.go

```diff
@@ -86,6 +95,9 @@ var (
 	useRemoteCerts = env.RegisterBoolVar("USE_REMOTE_CERTS", false,
 		"Whether to try to load CA certs from config Kubernetes cluster. Used for external Istiod.")
 
```

Comment:
Can we potentially infer this from the fields on the secret rather than adding a new variable?

Reply:
The secret isnt read by istiod, it just looks for the files in their correct spot so currently there is no way to see where the files are being mounted from.

Reply:
I am not sure that means we cannot infer it though? If we see the files, we can use them?

Reply:
To clarify - I agree we should detect

Reply:
Added a small detection for the tls.crt otherwise default to the istio file format

https://github.com/istio/istio/pull/39732/files#diff-0e039728d940bd528569ae3684ba2c8828df0f5d77456e3394fb0c94ea6e72e2R244-R271